core/trino-main/src/main/java/io/trino/SystemSessionProperties.java

```diff
@@ -506,6 +507,11 @@ public SystemSessionProperties(
                         "Enable a stats-based rule adding exchanges below GroupId",
                         optimizerConfig.isEnableForcedExchangeBelowGroupId(),
                         true),
+                booleanProperty(
```

Comment:
let's not add a flag for a rule that is already pretty sophisticated and we were thinking about removing it.